ui/src/components/Bouts/SpectrogramTimeline.tsx

```diff
@@ -161,7 +163,9 @@ export default function SpectrogramTimeline({
         spectrogramWindow: spectrogramWindow.current,
       }),
     );
-  }, [boutStartTime, boutEndTime]);
+    // Only run on initial load
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
```

Comment:
Probably not ideal to be doing this but I think this case in particular is pretty annoying to resolve since you need the DOM to render first… I believe the solution is a [callback ref](https://tkdodo.eu/blog/avoiding-use-effect-with-callback-refs) but totally fine not worrying about it for now…